### ISSUE TITLE
Crash after clean install

### ISSUE THREAD

**Issue by JesperWe** [nephila/djangocms-rest-view#2]
I have a clean new install of Django CMS (python 2.7, Django 1.8.14), running a dev server with `python manage.py runserver`
I followed your install instructions exactly.

When I add the two url lines to `urls.py` and try to access the CMS I now get:

```
Traceback (most recent call last):
  File "/usr/lib/python2.7/wsgiref/handlers.py", line 85, in run
    self.result = application(self.environ, self.start_response)
  File "/var/www/django/env/local/lib/python2.7/site-packages/django/contrib/staticfiles/handlers.py", line 63, in __call__
    return self.application(environ, start_response)
  File "/var/www/django/env/local/lib/python2.7/site-packages/django/core/handlers/wsgi.py", line 170, in __call__
    self.load_middleware()
  File "/var/www/django/env/local/lib/python2.7/site-packages/django/core/handlers/base.py", line 52, in load_middleware
    mw_instance = mw_class()
  File "/var/www/django/env/local/lib/python2.7/site-packages/django/middleware/locale.py", line 24, in __init__
    for url_pattern in get_resolver(None).url_patterns:
  File "/var/www/django/env/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 401, in url_patterns
    patterns = getattr(self.urlconf_module, "urlpatterns", self.urlconf_module)
  File "/var/www/django/env/local/lib/python2.7/site-packages/django/core/urlresolvers.py", line 395, in urlconf_module
    self._urlconf_module = import_module(self.urlconf_name)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/var/www/django/wmm/wmm/urls.py", line 17, in <module>
    url(r'^api/', include('djangocms_rest_view.urls')),
  File "/var/www/django/env/local/lib/python2.7/site-packages/django/conf/urls/__init__.py", line 33, in include
    urlconf_module = import_module(urlconf_module)
  File "/usr/lib/python2.7/importlib/__init__.py", line 37, in import_module
    __import__(name)
  File "/var/www/django/env/local/lib/python2.7/site-packages/djangocms_rest_view/urls.py", line 5, in <module>
    from .views import PageViewSet
  File "/var/www/django/env/local/lib/python2.7/site-packages/djangocms_rest_view/views.py", line 12, in <module>
    from .serializers import (
  File "/var/www/django/env/local/lib/python2.7/site-packages/djangocms_rest_view/serializers.py", line 9, in <module>
    from cms.plugin_rendering import render_placeholder
ImportError: cannot import name render_placeholder
```

**Comment by yakky** [comment]
@JesperWe djangocms-rest-view is currently tested on django CMS 3.2 and 3.3, no 3.4 compatibility yet

**Comment by yakky** [comment]
@JesperWe I just merged django CMS  3.4 compatibility, I should release compatible version later today, thanks for reporting
